src/components/SelectComponents/Autosuggest/__tests__/Autosuggest.spec.tsx

```diff
@@ -6,7 +6,8 @@ import { Autosuggest } from '../Autosuggest';
 import {
     SUGGESTIONS,
     SUGGESTION_TO_STRING,
-} from '../../../AutosuggestDeprecated/__mocks__/suggestions';
+    SuggestionsType,
+} from '../../../../constants/suggestions';
```

Comment:
It's not the application constants. Let's move `suggestions` to `__mocks__` folder for the component,

Reply:
:+1: